test/e2e-upgrade-tests.sh

```diff
@@ -42,8 +42,7 @@ function stage_test_resources() {
 # Skip installing istio as an add-on.
 # Skip installing a pvc as it is not used in upgrade tests
 # Skip installing a resource quota as it is not used in upgrade tests
-PVC=0 QUOTA=0 initialize "$@" --skip-istio-addon  --min-nodes=4 --max-nodes=4 --cluster-version=1.25 \
-  --install-latest-release
```

Comment:
I think this is why the update tests are failing?

Reply:
hmm, added it back in.